README.md

```diff
@@ -55,7 +55,7 @@ Parameters:
  * `extra`: ( optional ) string, Client provided metadata that get passed back in the callback.
```

Comment:
README needs update after the type change.

Reply:
fixed!